Sources/SwiftProtobuf/_MessageLayout.swift

```diff
@@ -182,7 +189,8 @@ import Foundation
         copyField: @escaping TrampolineCopier,
         areFieldsEqual: @escaping TrampolineEquater,
         performOnSubmessageStorage: @escaping SubmessageStoragePerformer,
-        performOnRawEnumValues: @escaping RawEnumValuesPerformer
+        performOnRawEnumValues: @escaping RawEnumValuesPerformer,
+        mapEntryLayout: @escaping MapEntryLayout
```

Comment:
Note to self - maybe look in the future about having a few versions of init that leaves off this one and then auto fills it with the stub.  i.e. - keep codegen size down by at the expense of a few more things in the runtime for common cases for things that aren't generally needed. (in theory the enum one might also be something to specialize without)